src/Builder/FormContractor.php

```diff
@@ -78,7 +79,11 @@ public function getFormFactory()
      */
     public function getFormBuilder($name, array $options = [])
     {
-        return $this->getFormFactory()->createNamedBuilder($name, 'form', null, $options);
+        // NEXT_MAJOR: Remove this line when drop Symfony <2.8 support
+        $formType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
+            ? FormType::class : 'form';
```

Comment:
don't add a check here, use always `FormType::class` this apply for all method_exists that will always return true on SF 2.8

Reply:
True, but shall I change every file or just the ones I've added? and then in another PR remove the rest.

Reply:
Do as you wish, as long as you make 2 commits